modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -48,7 +49,7 @@ sda.enkf.multisite <- function(settings, obs.mean, obs.cov, Q = NULL, restart=F,
   scalef <- settings$state.data.assimilation$scalef %>% as.numeric() # scale factor for localization
   var.names <- sapply(settings$state.data.assimilation$state.variable, '[[', "variable.name")
   names(var.names) <- NULL
-  multi.site.flag <- PEcAn.settings::is.MultiSettings(settings)
+  multi.site.flag <- T#PEcAn.settings::is.MultiSettings(settings)
```

Comment:
@bailsofhay is this a temporary change?

Reply:
@para2x told me to do that for some reason. I think I just forgot it was in there. It can go

Reply:
@serbinsh @bailsofhay I remember you guys told me that for whatever reason this breaks our code and I said we don't use this flag yet.

Reply:
I just removed it for now